app/src/molecules/InterventionModal/InterventionContent/InterventionInfo.tsx

```diff
@@ -39,23 +44,31 @@ export function InterventionInfo(props: InterventionInfoProps): JSX.Element {
           desktopStyle="bodyDefaultSemiBold"
           css={LINE_CLAMP_STYLE}
         >
-          {props.labwareName}
+          {props.labwareNickname ?? props.labwareName}
         </StyledText>
-        {props.labwareNickname != null ? (
+        {props.subtext != null ? (
           <StyledText
             oddStyle="hidden"
             desktopStyle="bodyDefaultRegular"
             color={COLORS.grey60}
             css={css`
               ${LINE_CLAMP_STYLE}
+              maring-bottom: 0.25rem
```

Comment:
```suggestion
              margin-bottom: 0.25rem
```
?
also is there a design system constant for `0.25rem`